packages/st2/debian/st2.st2auth.init

```diff
@@ -71,6 +71,8 @@ do_start()
   start-stop-daemon --start --quiet --chuid $RUNAS_USER:$RUNAS_GROUP --umask $UMASK --background --pidfile $PIDFILE \
      --exec $DAEMON -- $DAEMON_ARGS \
     || return 2
+  # wait for st2auth readiness to serve the requests
+  timeout 5 bash -c 'until echo > /dev/tcp/127.0.0.1/9100; do sleep 0.5; done' || return 2
```

Comment:
I'm having some issues with st2 e2e tests failing, and it looks like to be related to this change (st2auth service doesn't start at all causing tests to fail).

Will dig into it and try to increase the timeout, 5 seconds might simply be too little to give service enough time to connect to MongoDB and RabbitMQ.